src/track-menu-item.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { recordTracksEvent } from '@automattic/calypso-analytics';
```

Comment:
@chihsuan I'm wondering if the `calypso-analytics` package is really required? I'm not seeing any difference with the end result, the tracks look the same as what we have embedded in WooCommerce, `wc.tracks.recordEvent`.

What do you think?

Reply:
AFAIK, `wc.tracks.recordEvent` [adds a `wcadmin_` prefix to the track name](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/tracks/class-wc-site-tracking.php#L121).